docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+### Usage: /docker-entrypoint.sh <command> <args>
+command=$1
+
+case "$command" in
+  serve)
+    for file_candidate in $@ ; do
+      if test -f "$file_candidate"; then
+        tmpfile=$(mktemp /tmp/dex.config.yaml-XXXXXX)
+        gomplate -f "$file_candidate" -o "$tmpfile"
+        echo "config rendered successfully into the tmp file ${tmpfile}"
```

Comment:
Generally I don't think writing output from the entrypoint is not a good idea. If your logging system expects JSON logs this can mess with your log parser/analyzer and it doesn't really have any added value IMHO.

Reply:
I was not 100% sure about adding this `echo`. The main reason for it is that in case of an error in config Dex returns an error about some `/tmp/dex-config.yaml-ABCDEFG` file. This can be confusing for users.

Ok, let's get rid of this `echo`.